examples/flutter_firebase_login/lib/login/bloc/login_bloc.dart

```diff
@@ -36,9 +36,9 @@ class LoginBloc extends Bloc<LoginEvent, LoginState> {
 
   @override
   Stream<LoginState> mapEventToState(LoginEvent event) async* {
-    if (event is EmailChanged) {
+    if (event is LoginEmailChanged) {
       yield* _mapEmailChangedToState(event.email);
-    } else if (event is PasswordChanged) {
+    } else if (event is LoginPasswordChanged) {
       yield* _mapPasswordChangedToState(event.password);
     } else if (event is LoginWithGooglePressed) {
       yield* _mapLoginWithGooglePressedToState();
```

Comment:
can we also rename these helper functions to reflect the new event names?
`_mapLoginEmailChangedToState`, etc...